bindings/js/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "libredwg",
+  "version": "0.0.0-dev.0",
+  "description": "GNU LibreDWG is a free library to handle DWG/DXF files",
+  "main": "index.js",
+  "repository": "git://fr-an-k/libredwg.git",
```

Comment:
Really?

Reply:
it's just a test through my fork and reservation of the library name, of course I will transfer ownership if desired. I just need auto updates in my production environment.

Reply:
It's not a valid Url mostly. Thee domain is missing